addons/helm-revision-pruner/templates/cronjob.yaml

```diff
@@ -0,0 +1,191 @@
+{{- if or (ne .Values.image.repository "public.ecr.aws/o1j4x7p4/hello-porter-job") (.Values.global) }}
+{{- if $.Capabilities.APIVersions.Has "batch/v1/CronJob" }}
+apiVersion: batch/v1
+{{- else }}
+apiVersion: batch/v1beta1
+{{- end }}
+kind: CronJob
+metadata:
+  name: {{ include "docker-template.fullname" . }}
+  labels:
+    {{- include "docker-template.labels" . | nindent 4 }}
+    {{- if .Values.labels }}
+    {{- toYaml .Values.labels | nindent 4 }}
+    {{- end }}
+spec:
+  successfulJobsHistoryLimit: {{ .Values.schedule.successfulHistory }}
+  failedJobsHistoryLimit: {{ .Values.schedule.failedHistory }}
+  concurrencyPolicy: {{ ternary "Allow" "Forbid" .Values.allowConcurrency }}
+  startingDeadlineSeconds: 20
+  suspend: {{ not .Values.schedule.enabled }}
+  schedule: "{{ .Values.schedule.value }}"
+  jobTemplate:
+    metadata:
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        app.kubernetes.io/version: {{ .Chart.AppVersion }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        helm.sh/revision: "{{ .Release.Revision }}"
+        meta.helm.sh/release-name: "{{ .Release.Name }}"
+        {{- if .Values.stack.enabled }}
+        porter.run/stack-name: "{{ .Values.stack.name }}"
+        {{- end }}
+        porter.run/application-name: "{{ .Release.Name }}"
+        {{- if .Values.labels }}
+        {{- toYaml .Values.labels | nindent 8 }}
+        {{- end }}
+    spec:
+      template:
+        metadata:
+          annotations:
+            {{- if .Values.safeToEvict }}
+            "cluster-autoscaler.kubernetes.io/safe-to-evict": "true"
+            {{- else }}
+            "cluster-autoscaler.kubernetes.io/safe-to-evict": "false"
+            {{- end }}
+          labels:
+            {{- if .Values.podLabels }}
+            {{- toYaml .Values.podLabels | nindent 12 }}
+            {{- end }}
+        spec:
+          {{- if .Values.serviceAccount.create }}
+          serviceAccountName: cronjob-{{ include "docker-template.serviceAccountName" . }}
+          {{ else }}
+          serviceAccountName: {{ include "docker-template.serviceAccountName" . }}
+          {{- end }}
+          containers:
+          - name: {{ .Chart.Name }}
+            {{- if .Values.global }}
+            {{- if .Values.global.image }}
+            image: "{{ .Values.global.image.repository }}:{{ .Values.global.image.tag }}"
+            {{- end }}
+            {{- else }}
+            image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+            {{- end }}
+            imagePullPolicy: {{ .Values.image.pullPolicy }}
+            {{- if .Values.global }}
+            {{- if .Values.global.image }}
+            {{- if and .Values.container.command (not (eq .Values.global.image.repository "public.ecr.aws/o1j4x7p4/hello-porter")) }}
```

Comment:
should all this logic about hello-porter image be here if the base image is `public.ecr.aws/porter-sre/helm-revision-pruner`?

Reply:
Yeah good point - I wanted to get the chart out asap so reused a lot of values from our job chart.

Reply:
[1875111](https://github.com/porter-dev/porter-charts/pull/792/commits/1875111140baf2cc5a7e9c831d0d07d6f89cdf8a)